pytket/pytket/circuit/__init__.py

```diff
@@ -160,6 +160,118 @@ def add_wasm_to_reg(  # noqa: PLR0913
 setattr(Circuit, "add_wasm_to_reg", add_wasm_to_reg)  # noqa: B010
 
 
+def set_rng_seed(self: Circuit, creg: BitRegister, **kwargs: Any) -> Circuit:
```

Comment:
Have you thought about adding a second function taking a bit list instead of `BitRegister` ?